### ISSUE TITLE
[4.0] tinyMCE - unwanted CMS Content dropdown button

### ISSUE THREAD

**Issue by schnuti** [joomla/joomla-cms#21711]
### Steps to reproduce the issue
If you have a form with a "custom" editor field you see it.
Otherwise edit the article form in administrator/components/com_content/forms/article.xml
- Change buttons="true" to buttons="false" for field-name articletext
- Open New Article
- The CMS Content dropdown button is there with an empty dropdown list.

![tiny_mce_cms_button](https://user-images.githubusercontent.com/956537/44307173-06db5d80-a39e-11e8-84c8-dd56f3ab0f7a.jpg)
Example:

          <field
             name="descripton" 
             type="editor" 
             ==>    buttons="false" 
             filter="JComponentHelper::filterText"
             height="200" 
             label="COM_MYCOMP_FIELD_DESCRIPPTION_LABEL"/>

### Expected result
No button 

### Actual result
Button!

### Comment
The height set in the form field above is of no use as it is overridden by a system wide parameter set in the tinyMCE plugin. That behaviour should be removed! I have to use a system plugin to change to the expected height.

**Comment by brianteeman** [comment]
This a js issue in build\media_src\editors\tinymce\js\tinymce.es6.js 

The button is always created even if you don't have any buttons enabled. Sorry but I have no idea how to fix that in js

**Comment by dgrammatiko** [comment]
@brianteeman this should fix it:
```js
      if (buttonValues.length) {
        options.setup = (editor) => {
          editor.addButton('jxtdbuttons', {
            type: 'menubutton',
            text: Joomla.JText._('PLG_TINY_CORE_BUTTONS'),
            icon: 'none icon-joomla',
            menu: buttonValues,
          });
        };
      }
```
Line 85 onwards

**Comment by brianteeman** [comment]
Closed see #21725
